CHANGES/7265.feature

```diff
@@ -0,0 +1 @@
+The unused size parameter in the feed_data method has been removed
```

Comment:
It's not a feature but a breaking change in the public API, probably.

Reply:
So, I should use `7265.removal`?